CRITICAL - 2023-10-25 05:20:51 --> "id_unique" is not a valid rule.
in SYSTEMPATH\Validation\Validation.php on line 317.
 1 SYSTEMPATH\Validation\Validation.php(317): CodeIgniter\Validation\Exceptions\ValidationException::forRuleNotFound('id_unique')
 2 SYSTEMPATH\Validation\Validation.php(195): CodeIgniter\Validation\Validation->processRules('email', 'email', 'k@gmail.com', [...], [...])
 3 SYSTEMPATH\Controller.php(150): CodeIgniter\Validation\Validation->run()
 4 APPPATH\Controllers\Register.php(34): CodeIgniter\Controller->validate([...])
 5 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Register->index()
 6 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 7 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-25 06:43:50 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'ci4', 3306, '', 0)
#1 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\codig\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\codig\system\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codig\system\Validation\Validation.php(309): CodeIgniter\Validation\StrictRules\Rules->is_unique('ksajaykujmar73@...', 'email', Array, NULL, 'email')
#6 C:\xampp\htdocs\codig\system\Validation\Validation.php(195): CodeIgniter\Validation\Validation->processRules('email', 'email', 'ksajaykujmar73@...', Array, Array)
#7 C:\xampp\htdocs\codig\system\Controller.php(150): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\codig\app\Controllers\Register.php(35): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\codig\system\CodeIgniter.php(919): App\Controllers\Register->index()
#10 C:\xampp\htdocs\codig\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#11 C:\xampp\htdocs\codig\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\codig\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\codig\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 C:\xampp\htdocs\codig\system\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\codig\system\Validation\Validation.php(309): CodeIgniter\Validation\StrictRules\Rules->is_unique('ksajaykujmar73@...', 'email', Array, NULL, 'email')
#5 C:\xampp\htdocs\codig\system\Validation\Validation.php(195): CodeIgniter\Validation\Validation->processRules('email', 'email', 'ksajaykujmar73@...', Array, Array)
#6 C:\xampp\htdocs\codig\system\Controller.php(150): CodeIgniter\Validation\Validation->run()
#7 C:\xampp\htdocs\codig\app\Controllers\Register.php(35): CodeIgniter\Controller->validate(Array)
#8 C:\xampp\htdocs\codig\system\CodeIgniter.php(919): App\Controllers\Register->index()
#9 C:\xampp\htdocs\codig\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#10 C:\xampp\htdocs\codig\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\codig\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-25 06:43:50 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Validation\Validation.php(309): CodeIgniter\Validation\StrictRules\Rules->is_unique('ksajaykujmar73@gmail.com', 'email', [...], null, 'email')
 5 SYSTEMPATH\Validation\Validation.php(195): CodeIgniter\Validation\Validation->processRules('email', 'email', 'ksajaykujmar73@gmail.com', [...], [...])
 6 SYSTEMPATH\Controller.php(150): CodeIgniter\Validation\Validation->run()
 7 APPPATH\Controllers\Register.php(35): CodeIgniter\Controller->validate([...])
 8 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Register->index()
 9 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
10 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-25 06:45:08 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'ci4', 3306, '', 0)
#1 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\codig\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\codig\system\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codig\system\Validation\Validation.php(309): CodeIgniter\Validation\StrictRules\Rules->is_unique('ksajaykujmar73@...', 'email', Array, NULL, 'email')
#6 C:\xampp\htdocs\codig\system\Validation\Validation.php(195): CodeIgniter\Validation\Validation->processRules('email', 'email', 'ksajaykujmar73@...', Array, Array)
#7 C:\xampp\htdocs\codig\system\Controller.php(150): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\codig\app\Controllers\Register.php(35): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\codig\system\CodeIgniter.php(919): App\Controllers\Register->index()
#10 C:\xampp\htdocs\codig\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#11 C:\xampp\htdocs\codig\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\codig\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\codig\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 C:\xampp\htdocs\codig\system\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\codig\system\Validation\Validation.php(309): CodeIgniter\Validation\StrictRules\Rules->is_unique('ksajaykujmar73@...', 'email', Array, NULL, 'email')
#5 C:\xampp\htdocs\codig\system\Validation\Validation.php(195): CodeIgniter\Validation\Validation->processRules('email', 'email', 'ksajaykujmar73@...', Array, Array)
#6 C:\xampp\htdocs\codig\system\Controller.php(150): CodeIgniter\Validation\Validation->run()
#7 C:\xampp\htdocs\codig\app\Controllers\Register.php(35): CodeIgniter\Controller->validate(Array)
#8 C:\xampp\htdocs\codig\system\CodeIgniter.php(919): App\Controllers\Register->index()
#9 C:\xampp\htdocs\codig\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#10 C:\xampp\htdocs\codig\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\codig\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-25 06:45:08 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Validation\Validation.php(309): CodeIgniter\Validation\StrictRules\Rules->is_unique('ksajaykujmar73@gmail.com', 'email', [...], null, 'email')
 5 SYSTEMPATH\Validation\Validation.php(195): CodeIgniter\Validation\Validation->processRules('email', 'email', 'ksajaykujmar73@gmail.com', [...], [...])
 6 SYSTEMPATH\Controller.php(150): CodeIgniter\Validation\Validation->run()
 7 APPPATH\Controllers\Register.php(35): CodeIgniter\Controller->validate([...])
 8 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Register->index()
 9 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
10 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-25 06:50:10 --> mysqli_sql_exception: Table 'ci4.users' doesn't exist in engine in C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\codig\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\codig\system\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codig\system\Validation\Validation.php(309): CodeIgniter\Validation\StrictRules\Rules->is_unique('ksajaykujmar73@...', 'email', Array, NULL, 'email')
#6 C:\xampp\htdocs\codig\system\Validation\Validation.php(195): CodeIgniter\Validation\Validation->processRules('email', 'email', 'ksajaykujmar73@...', Array, Array)
#7 C:\xampp\htdocs\codig\system\Controller.php(150): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\codig\app\Controllers\Register.php(35): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\codig\system\CodeIgniter.php(919): App\Controllers\Register->index()
#10 C:\xampp\htdocs\codig\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#11 C:\xampp\htdocs\codig\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\codig\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-10-25 06:50:10 --> Table 'ci4.users' doesn't exist in engine
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\Validation\Validation.php(309): CodeIgniter\Validation\StrictRules\Rules->is_unique('ksajaykujmar73@gmail.com', 'email', [...], null, 'email')
 4 SYSTEMPATH\Validation\Validation.php(195): CodeIgniter\Validation\Validation->processRules('email', 'email', 'ksajaykujmar73@gmail.com', [...], [...])
 5 SYSTEMPATH\Controller.php(150): CodeIgniter\Validation\Validation->run()
 6 APPPATH\Controllers\Register.php(35): CodeIgniter\Controller->validate([...])
 7 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Register->index()
 8 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-25 08:59:49 --> mysqli_sql_exception: Table 'ci4.users' doesn't exist in engine in C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\codig\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\codig\system\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codig\system\Validation\Validation.php(309): CodeIgniter\Validation\StrictRules\Rules->is_unique('g@gmail.com', 'email', Array, NULL, 'email')
#6 C:\xampp\htdocs\codig\system\Validation\Validation.php(195): CodeIgniter\Validation\Validation->processRules('email', 'email', 'g@gmail.com', Array, Array)
#7 C:\xampp\htdocs\codig\system\Controller.php(150): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\codig\app\Controllers\Register.php(35): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\codig\system\CodeIgniter.php(919): App\Controllers\Register->index()
#10 C:\xampp\htdocs\codig\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#11 C:\xampp\htdocs\codig\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\codig\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-10-25 08:59:49 --> Table 'ci4.users' doesn't exist in engine
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\Validation\Validation.php(309): CodeIgniter\Validation\StrictRules\Rules->is_unique('g@gmail.com', 'email', [...], null, 'email')
 4 SYSTEMPATH\Validation\Validation.php(195): CodeIgniter\Validation\Validation->processRules('email', 'email', 'g@gmail.com', [...], [...])
 5 SYSTEMPATH\Controller.php(150): CodeIgniter\Validation\Validation->run()
 6 APPPATH\Controllers\Register.php(35): CodeIgniter\Controller->validate([...])
 7 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Register->index()
 8 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-25 09:10:26 --> mysqli_sql_exception: Table 'ci4.users' doesn't exist in engine in C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\codig\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\codig\system\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codig\system\Validation\Validation.php(309): CodeIgniter\Validation\StrictRules\Rules->is_unique('g@gmail.com', 'email', Array, NULL, 'email')
#6 C:\xampp\htdocs\codig\system\Validation\Validation.php(195): CodeIgniter\Validation\Validation->processRules('email', 'email', 'g@gmail.com', Array, Array)
#7 C:\xampp\htdocs\codig\system\Controller.php(150): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\codig\app\Controllers\Register.php(35): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\codig\system\CodeIgniter.php(919): App\Controllers\Register->index()
#10 C:\xampp\htdocs\codig\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#11 C:\xampp\htdocs\codig\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\codig\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-10-25 09:10:26 --> Table 'ci4.users' doesn't exist in engine
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\Validation\Validation.php(309): CodeIgniter\Validation\StrictRules\Rules->is_unique('g@gmail.com', 'email', [...], null, 'email')
 4 SYSTEMPATH\Validation\Validation.php(195): CodeIgniter\Validation\Validation->processRules('email', 'email', 'g@gmail.com', [...], [...])
 5 SYSTEMPATH\Controller.php(150): CodeIgniter\Validation\Validation->run()
 6 APPPATH\Controllers\Register.php(35): CodeIgniter\Controller->validate([...])
 7 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Register->index()
 8 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-25 09:16:26 --> mysqli_sql_exception: Unknown column 'username' in 'field list' in C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codig\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codig\app\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\codig\app\Controllers\Register.php(43): App\Models\Register_model->createUser(Array)
#6 C:\xampp\htdocs\codig\system\CodeIgniter.php(919): App\Controllers\Register->index()
#7 C:\xampp\htdocs\codig\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#8 C:\xampp\htdocs\codig\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\codig\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-25 09:16:26 --> Unknown column 'username' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `pass`, `uniid`, `activation_date`) VALUES (:username:, :pass:, :uniid:, :activation_date:)', [...], false)
 2 APPPATH\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Register.php(43): App\Models\Register_model->createUser([...])
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Register->index()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-25 09:17:30 --> mysqli_sql_exception: Unknown column 'username' in 'field list' in C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codig\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codig\app\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\codig\app\Controllers\Register.php(43): App\Models\Register_model->createUser(Array)
#6 C:\xampp\htdocs\codig\system\CodeIgniter.php(919): App\Controllers\Register->index()
#7 C:\xampp\htdocs\codig\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#8 C:\xampp\htdocs\codig\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\codig\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-25 09:17:30 --> Unknown column 'username' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `pass`, `uniid`, `activation_date`) VALUES (:username:, :pass:, :uniid:, :activation_date:)', [...], false)
 2 APPPATH\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Register.php(43): App\Models\Register_model->createUser([...])
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Register->index()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-25 09:17:55 --> mysqli_sql_exception: Unknown column 'username' in 'field list' in C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codig\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codig\app\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\codig\app\Controllers\Register.php(43): App\Models\Register_model->createUser(Array)
#6 C:\xampp\htdocs\codig\system\CodeIgniter.php(919): App\Controllers\Register->index()
#7 C:\xampp\htdocs\codig\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#8 C:\xampp\htdocs\codig\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\codig\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-25 09:17:55 --> Unknown column 'username' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `pass`, `uniid`, `activation_date`) VALUES (:username:, :pass:, :uniid:, :activation_date:)', [...], false)
 2 APPPATH\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Register.php(43): App\Models\Register_model->createUser([...])
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Register->index()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-25 09:21:28 --> mysqli_sql_exception: Unknown column 'pass' in 'field list' in C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codig\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codig\app\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\codig\app\Controllers\Register.php(43): App\Models\Register_model->createUser(Array)
#6 C:\xampp\htdocs\codig\system\CodeIgniter.php(919): App\Controllers\Register->index()
#7 C:\xampp\htdocs\codig\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#8 C:\xampp\htdocs\codig\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\codig\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-25 09:21:28 --> Unknown column 'pass' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `pass`, `uniid`, `activation_date`) VALUES (:username:, :pass:, :uniid:, :activation_date:)', [...], false)
 2 APPPATH\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Register.php(43): App\Models\Register_model->createUser([...])
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Register->index()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-25 09:22:24 --> mysqli_sql_exception: Unknown column 'uniid' in 'field list' in C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codig\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codig\app\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\codig\app\Controllers\Register.php(43): App\Models\Register_model->createUser(Array)
#6 C:\xampp\htdocs\codig\system\CodeIgniter.php(919): App\Controllers\Register->index()
#7 C:\xampp\htdocs\codig\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#8 C:\xampp\htdocs\codig\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\codig\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-25 09:22:24 --> Unknown column 'uniid' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `pass`, `uniid`, `activation_date`) VALUES (:username:, :pass:, :uniid:, :activation_date:)', [...], false)
 2 APPPATH\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Register.php(43): App\Models\Register_model->createUser([...])
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Register->index()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-25 09:24:14 --> mysqli_sql_exception: Duplicate entry '0' for key 'PRIMARY' in C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codig\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codig\app\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\codig\app\Controllers\Register.php(43): App\Models\Register_model->createUser(Array)
#6 C:\xampp\htdocs\codig\system\CodeIgniter.php(919): App\Controllers\Register->index()
#7 C:\xampp\htdocs\codig\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#8 C:\xampp\htdocs\codig\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\codig\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-25 09:24:14 --> Duplicate entry '0' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `pass`, `uniid`, `activation_date`) VALUES (:username:, :pass:, :uniid:, :activation_date:)', [...], false)
 2 APPPATH\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Register.php(43): App\Models\Register_model->createUser([...])
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Register->index()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-25 09:24:28 --> mysqli_sql_exception: Duplicate entry '0' for key 'PRIMARY' in C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codig\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codig\app\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\codig\app\Controllers\Register.php(43): App\Models\Register_model->createUser(Array)
#6 C:\xampp\htdocs\codig\system\CodeIgniter.php(919): App\Controllers\Register->index()
#7 C:\xampp\htdocs\codig\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#8 C:\xampp\htdocs\codig\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\codig\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-25 09:24:28 --> Duplicate entry '0' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `pass`, `uniid`, `activation_date`) VALUES (:username:, :pass:, :uniid:, :activation_date:)', [...], false)
 2 APPPATH\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Register.php(43): App\Models\Register_model->createUser([...])
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Register->index()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-25 09:24:37 --> mysqli_sql_exception: Duplicate entry '0' for key 'PRIMARY' in C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codig\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codig\app\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\codig\app\Controllers\Register.php(43): App\Models\Register_model->createUser(Array)
#6 C:\xampp\htdocs\codig\system\CodeIgniter.php(919): App\Controllers\Register->index()
#7 C:\xampp\htdocs\codig\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#8 C:\xampp\htdocs\codig\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\codig\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-25 09:24:37 --> Duplicate entry '0' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `pass`, `uniid`, `activation_date`) VALUES (:username:, :pass:, :uniid:, :activation_date:)', [...], false)
 2 APPPATH\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Register.php(43): App\Models\Register_model->createUser([...])
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Register->index()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-25 09:25:10 --> mysqli_sql_exception: Duplicate entry '0' for key 'PRIMARY' in C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codig\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codig\app\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\codig\app\Controllers\Register.php(43): App\Models\Register_model->createUser(Array)
#6 C:\xampp\htdocs\codig\system\CodeIgniter.php(919): App\Controllers\Register->index()
#7 C:\xampp\htdocs\codig\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#8 C:\xampp\htdocs\codig\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\codig\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-25 09:25:10 --> Duplicate entry '0' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `pass`, `uniid`, `activation_date`) VALUES (:username:, :pass:, :uniid:, :activation_date:)', [...], false)
 2 APPPATH\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Register.php(43): App\Models\Register_model->createUser([...])
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Register->index()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-25 09:26:08 --> mysqli_sql_exception: Duplicate entry '0' for key 'PRIMARY' in C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codig\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codig\app\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\codig\app\Controllers\Register.php(43): App\Models\Register_model->createUser(Array)
#6 C:\xampp\htdocs\codig\system\CodeIgniter.php(919): App\Controllers\Register->index()
#7 C:\xampp\htdocs\codig\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#8 C:\xampp\htdocs\codig\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\codig\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-25 09:26:08 --> Duplicate entry '0' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `pass`, `uniid`, `activation_date`) VALUES (:username:, :pass:, :uniid:, :activation_date:)', [...], false)
 2 APPPATH\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Register.php(43): App\Models\Register_model->createUser([...])
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Register->index()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-25 09:27:11 --> mysqli_sql_exception: Duplicate entry '' for key 'email' in C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codig\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codig\app\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\codig\app\Controllers\Register.php(43): App\Models\Register_model->createUser(Array)
#6 C:\xampp\htdocs\codig\system\CodeIgniter.php(919): App\Controllers\Register->index()
#7 C:\xampp\htdocs\codig\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#8 C:\xampp\htdocs\codig\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\codig\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-25 09:27:11 --> Duplicate entry '' for key 'email'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `pass`, `uniid`, `activation_date`) VALUES (:username:, :pass:, :uniid:, :activation_date:)', [...], false)
 2 APPPATH\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Register.php(43): App\Models\Register_model->createUser([...])
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Register->index()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-25 09:27:19 --> mysqli_sql_exception: Duplicate entry '' for key 'email' in C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codig\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codig\app\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\codig\app\Controllers\Register.php(43): App\Models\Register_model->createUser(Array)
#6 C:\xampp\htdocs\codig\system\CodeIgniter.php(919): App\Controllers\Register->index()
#7 C:\xampp\htdocs\codig\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#8 C:\xampp\htdocs\codig\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\codig\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-25 09:27:19 --> Duplicate entry '' for key 'email'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `pass`, `uniid`, `activation_date`) VALUES (:username:, :pass:, :uniid:, :activation_date:)', [...], false)
 2 APPPATH\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Register.php(43): App\Models\Register_model->createUser([...])
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Register->index()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-25 09:27:34 --> mysqli_sql_exception: Duplicate entry '' for key 'email' in C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codig\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codig\app\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\codig\app\Controllers\Register.php(43): App\Models\Register_model->createUser(Array)
#6 C:\xampp\htdocs\codig\system\CodeIgniter.php(919): App\Controllers\Register->index()
#7 C:\xampp\htdocs\codig\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#8 C:\xampp\htdocs\codig\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\codig\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-25 09:27:34 --> Duplicate entry '' for key 'email'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `pass`, `uniid`, `activation_date`) VALUES (:username:, :pass:, :uniid:, :activation_date:)', [...], false)
 2 APPPATH\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Register.php(43): App\Models\Register_model->createUser([...])
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Register->index()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-25 09:27:44 --> mysqli_sql_exception: Duplicate entry '' for key 'email' in C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codig\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codig\app\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\codig\app\Controllers\Register.php(43): App\Models\Register_model->createUser(Array)
#6 C:\xampp\htdocs\codig\system\CodeIgniter.php(919): App\Controllers\Register->index()
#7 C:\xampp\htdocs\codig\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#8 C:\xampp\htdocs\codig\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\codig\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-25 09:27:44 --> Duplicate entry '' for key 'email'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `pass`, `uniid`, `activation_date`) VALUES (:username:, :pass:, :uniid:, :activation_date:)', [...], false)
 2 APPPATH\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Register.php(43): App\Models\Register_model->createUser([...])
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Register->index()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-25 09:28:44 --> mysqli_sql_exception: Duplicate entry '' for key 'email' in C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codig\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codig\app\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\codig\app\Controllers\Register.php(43): App\Models\Register_model->createUser(Array)
#6 C:\xampp\htdocs\codig\system\CodeIgniter.php(919): App\Controllers\Register->index()
#7 C:\xampp\htdocs\codig\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#8 C:\xampp\htdocs\codig\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\codig\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-25 09:28:44 --> Duplicate entry '' for key 'email'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `pass`, `uniid`, `activation_date`) VALUES (:username:, :pass:, :uniid:, :activation_date:)', [...], false)
 2 APPPATH\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Register.php(43): App\Models\Register_model->createUser([...])
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Register->index()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-25 09:31:45 --> mysqli_sql_exception: Table 'ci4.users' doesn't exist in engine in C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\codig\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\codig\system\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codig\system\Validation\Validation.php(309): CodeIgniter\Validation\StrictRules\Rules->is_unique('ksajlaykumapppp...', 'email', Array, NULL, 'email')
#6 C:\xampp\htdocs\codig\system\Validation\Validation.php(195): CodeIgniter\Validation\Validation->processRules('email', 'email', 'ksajlaykumapppp...', Array, Array)
#7 C:\xampp\htdocs\codig\system\Controller.php(150): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\codig\app\Controllers\Register.php(35): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\codig\system\CodeIgniter.php(919): App\Controllers\Register->index()
#10 C:\xampp\htdocs\codig\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#11 C:\xampp\htdocs\codig\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\codig\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-10-25 09:31:45 --> Table 'ci4.users' doesn't exist in engine
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\Validation\Validation.php(309): CodeIgniter\Validation\StrictRules\Rules->is_unique('ksajlaykumapppppr73@gmail.com', 'email', [...], null, 'email')
 4 SYSTEMPATH\Validation\Validation.php(195): CodeIgniter\Validation\Validation->processRules('email', 'email', 'ksajlaykumapppppr73@gmail.com', [...], [...])
 5 SYSTEMPATH\Controller.php(150): CodeIgniter\Validation\Validation->run()
 6 APPPATH\Controllers\Register.php(35): CodeIgniter\Controller->validate([...])
 7 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Register->index()
 8 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-25 09:32:01 --> mysqli_sql_exception: Table 'ci4.users' doesn't exist in engine in C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\codig\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\codig\system\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codig\system\Validation\Validation.php(309): CodeIgniter\Validation\StrictRules\Rules->is_unique('h@ajayks.com', 'email', Array, NULL, 'email')
#6 C:\xampp\htdocs\codig\system\Validation\Validation.php(195): CodeIgniter\Validation\Validation->processRules('email', 'email', 'h@ajayks.com', Array, Array)
#7 C:\xampp\htdocs\codig\system\Controller.php(150): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\codig\app\Controllers\Register.php(35): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\codig\system\CodeIgniter.php(919): App\Controllers\Register->index()
#10 C:\xampp\htdocs\codig\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#11 C:\xampp\htdocs\codig\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\codig\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-10-25 09:32:01 --> Table 'ci4.users' doesn't exist in engine
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\Validation\Validation.php(309): CodeIgniter\Validation\StrictRules\Rules->is_unique('h@ajayks.com', 'email', [...], null, 'email')
 4 SYSTEMPATH\Validation\Validation.php(195): CodeIgniter\Validation\Validation->processRules('email', 'email', 'h@ajayks.com', [...], [...])
 5 SYSTEMPATH\Controller.php(150): CodeIgniter\Validation\Validation->run()
 6 APPPATH\Controllers\Register.php(35): CodeIgniter\Controller->validate([...])
 7 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Register->index()
 8 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-25 09:35:28 --> mysqli_sql_exception: Duplicate entry '' for key 'email' in C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codig\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codig\app\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\codig\app\Controllers\Register.php(44): App\Models\Register_model->createUser(Array)
#6 C:\xampp\htdocs\codig\system\CodeIgniter.php(919): App\Controllers\Register->index()
#7 C:\xampp\htdocs\codig\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#8 C:\xampp\htdocs\codig\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\codig\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-25 09:35:28 --> Duplicate entry '' for key 'email'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `pass`, `uniid`, `activation_date`) VALUES (:username:, :pass:, :uniid:, :activation_date:)', [...], false)
 2 APPPATH\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Register.php(44): App\Models\Register_model->createUser([...])
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Register->index()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-25 09:35:58 --> mysqli_sql_exception: Duplicate entry '' for key 'email' in C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codig\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codig\app\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\codig\app\Controllers\Register.php(44): App\Models\Register_model->createUser(Array)
#6 C:\xampp\htdocs\codig\system\CodeIgniter.php(919): App\Controllers\Register->index()
#7 C:\xampp\htdocs\codig\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#8 C:\xampp\htdocs\codig\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\codig\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-25 09:35:58 --> Duplicate entry '' for key 'email'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `pass`, `uniid`, `activation_date`) VALUES (:username:, :pass:, :uniid:, :activation_date:)', [...], false)
 2 APPPATH\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Register.php(44): App\Models\Register_model->createUser([...])
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Register->index()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-25 09:36:48 --> mysqli_sql_exception: Duplicate entry '' for key 'email' in C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codig\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codig\app\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\codig\app\Controllers\Register.php(44): App\Models\Register_model->createUser(Array)
#6 C:\xampp\htdocs\codig\system\CodeIgniter.php(919): App\Controllers\Register->index()
#7 C:\xampp\htdocs\codig\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#8 C:\xampp\htdocs\codig\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\codig\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-25 09:36:48 --> Duplicate entry '' for key 'email'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `pass`, `uniid`, `activation_date`) VALUES (:username:, :pass:, :uniid:, :activation_date:)', [...], false)
 2 APPPATH\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Register.php(44): App\Models\Register_model->createUser([...])
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Register->index()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-25 09:46:34 --> You must set the database table to be used with your query.
in SYSTEMPATH\Database\BaseConnection.php on line 883.
 1 SYSTEMPATH\Model.php(635): CodeIgniter\Database\BaseConnection->table(null)
 2 SYSTEMPATH\Model.php(863): CodeIgniter\Model->builder()
 3 APPPATH\Controllers\Register.php(38): CodeIgniter\Model->__call('isEmailUnique', [...])
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Register->index()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-25 09:46:47 --> You must set the database table to be used with your query.
in SYSTEMPATH\Database\BaseConnection.php on line 883.
 1 SYSTEMPATH\Model.php(635): CodeIgniter\Database\BaseConnection->table(null)
 2 SYSTEMPATH\Model.php(863): CodeIgniter\Model->builder()
 3 APPPATH\Controllers\Register.php(38): CodeIgniter\Model->__call('isEmailUnique', [...])
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Register->index()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-25 09:50:26 --> Cannot access protected property App\Models\Register_model::$table
in APPPATH\Controllers\Register.php on line 25.
 1 SYSTEMPATH\CodeIgniter.php(891): App\Controllers\Register->__construct()
 2 SYSTEMPATH\CodeIgniter.php(485): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-25 09:50:41 --> Cannot access protected property App\Models\Register_model::$table
in APPPATH\Controllers\Register.php on line 25.
 1 SYSTEMPATH\CodeIgniter.php(891): App\Controllers\Register->__construct()
 2 SYSTEMPATH\CodeIgniter.php(485): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-25 09:51:47 --> mysqli_sql_exception: Duplicate entry '' for key 'email' in C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\codig\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codig\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codig\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codig\app\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\codig\app\Controllers\Register.php(44): App\Models\Register_model->createUser(Array)
#6 C:\xampp\htdocs\codig\system\CodeIgniter.php(919): App\Controllers\Register->index()
#7 C:\xampp\htdocs\codig\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#8 C:\xampp\htdocs\codig\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\codig\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-25 09:51:47 --> Duplicate entry '' for key 'email'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `pass`, `uniid`, `activation_date`) VALUES (:username:, :pass:, :uniid:, :activation_date:)', [...], false)
 2 APPPATH\Models\Register_model.php(15): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Register.php(44): App\Models\Register_model->createUser([...])
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Register->index()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-25 18:32:23 --> Too few arguments to function App\Controllers\Regform::__construct(), 0 passed in C:\xampp\htdocs\clgproject\system\CodeIgniter.php on line 891 and exactly 1 expected
in APPPATH\Controllers\Regform.php on line 12.
 1 SYSTEMPATH\CodeIgniter.php(891): App\Controllers\Regform->__construct()
 2 SYSTEMPATH\CodeIgniter.php(485): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-25 18:33:06 --> Too few arguments to function App\Controllers\Regform::__construct(), 0 passed in C:\xampp\htdocs\clgproject\system\CodeIgniter.php on line 891 and exactly 1 expected
in APPPATH\Controllers\Regform.php on line 12.
 1 SYSTEMPATH\CodeIgniter.php(891): App\Controllers\Regform->__construct()
 2 SYSTEMPATH\CodeIgniter.php(485): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-25 18:37:52 --> mysqli_sql_exception: Table 'officeauto.user' doesn't exist in C:\xampp\htdocs\clgproject\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\clgproject\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\clgproject\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\clgproject\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\clgproject\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\clgproject\system\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\clgproject\system\Validation\Validation.php(309): CodeIgniter\Validation\StrictRules\Rules->is_unique('ksajaykumappr73...', 'email', Array, NULL, 'email')
#6 C:\xampp\htdocs\clgproject\system\Validation\Validation.php(195): CodeIgniter\Validation\Validation->processRules('email', 'email', 'ksajaykumappr73...', Array, Array)
#7 C:\xampp\htdocs\clgproject\system\Controller.php(150): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\clgproject\app\Controllers\Regform.php(26): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\clgproject\system\CodeIgniter.php(919): App\Controllers\Regform->index()
#10 C:\xampp\htdocs\clgproject\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Regform))
#11 C:\xampp\htdocs\clgproject\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\clgproject\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-10-25 18:37:52 --> Table 'officeauto.user' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `user`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\Validation\Validation.php(309): CodeIgniter\Validation\StrictRules\Rules->is_unique('ksajaykumappr73@gmail.com', 'email', [...], null, 'email')
 4 SYSTEMPATH\Validation\Validation.php(195): CodeIgniter\Validation\Validation->processRules('email', 'email', 'ksajaykumappr73@gmail.com', [...], [...])
 5 SYSTEMPATH\Controller.php(150): CodeIgniter\Validation\Validation->run()
 6 APPPATH\Controllers\Regform.php(26): CodeIgniter\Controller->validate([...])
 7 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Regform->index()
 8 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Regform))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-25 18:39:43 --> mysqli_sql_exception: Column 'blood' cannot be null in C:\xampp\htdocs\clgproject\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\clgproject\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `st...', 0)
#1 C:\xampp\htdocs\clgproject\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `st...')
#2 C:\xampp\htdocs\clgproject\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `st...')
#3 C:\xampp\htdocs\clgproject\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `st...', Array, false)
#4 C:\xampp\htdocs\clgproject\app\Models\Regform_model.php(16): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\clgproject\app\Controllers\Regform.php(91): App\Models\Regform_model->createUser(Array, Array)
#6 C:\xampp\htdocs\clgproject\system\CodeIgniter.php(919): App\Controllers\Regform->index()
#7 C:\xampp\htdocs\clgproject\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Regform))
#8 C:\xampp\htdocs\clgproject\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\clgproject\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-25 18:39:43 --> Column 'blood' cannot be null
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `stdbasicinfo2` (`sname`, `adno`, `regno`, `gender`, `dob`, `adyear`, `program`, `adress`, `nation`, `state1`, `district`, `pincode`, `adhaar`, `cast`, `religion`, `income`, `guardian`, `guardianrel`, `guardiancon`, `email1`, `phno`, `blood`) VALUES (:sname:, :adno:, :regno:, :gender:, :dob:, :adyear:, :program:, :adress:, :nation:, :state1:, :district:, :pincode:, :adhaar:, :cast:, :religion:, :income:, :guardian:, :guardianrel:, :guardiancon:, :email1:, :phno:, :blood:)', [...], false)
 2 APPPATH\Models\Regform_model.php(16): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Regform.php(91): App\Models\Regform_model->createUser([...], [...])
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Regform->index()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Regform))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-25 18:41:42 --> mysqli_sql_exception: Column 'act1' cannot be null in C:\xampp\htdocs\clgproject\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\clgproject\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `st...', 0)
#1 C:\xampp\htdocs\clgproject\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `st...')
#2 C:\xampp\htdocs\clgproject\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `st...')
#3 C:\xampp\htdocs\clgproject\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `st...', Array, false)
#4 C:\xampp\htdocs\clgproject\app\Models\Regform_model.php(17): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\clgproject\app\Controllers\Regform.php(91): App\Models\Regform_model->createUser(Array, Array)
#6 C:\xampp\htdocs\clgproject\system\CodeIgniter.php(919): App\Controllers\Regform->index()
#7 C:\xampp\htdocs\clgproject\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Regform))
#8 C:\xampp\htdocs\clgproject\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\clgproject\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-25 18:41:42 --> Column 'act1' cannot be null
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `stdmoreinfo2` (`sfather`, `smother`, `padres`, `fstate`, `mstate`, `pcontact`, `clgtoresdis`, `hsnm`, `hssyl`, `hsyr`, `hsmrk`, `hsperce`, `hssnm`, `hsssyl`, `hssyr`, `hssmrk`, `hssperce`, `act1`, `act2`, `act3`, `act4`) VALUES (:sfather:, :smother:, :padres:, :fstate:, :mstate:, :pcontact:, :clgtoresdis:, :hsnm:, :hssyl:, :hsyr:, :hsmrk:, :hsperce:, :hssnm:, :hsssyl:, :hssyr:, :hssmrk:, :hssperce:, :act1:, :act2:, :act3:, :act4:)', [...], false)
 2 APPPATH\Models\Regform_model.php(17): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Regform.php(91): App\Models\Regform_model->createUser([...], [...])
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Regform->index()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Regform))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-25 18:42:41 --> mysqli_sql_exception: Duplicate entry '225' for key 'PRIMARY' in C:\xampp\htdocs\clgproject\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\clgproject\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `st...', 0)
#1 C:\xampp\htdocs\clgproject\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `st...')
#2 C:\xampp\htdocs\clgproject\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `st...')
#3 C:\xampp\htdocs\clgproject\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `st...', Array, false)
#4 C:\xampp\htdocs\clgproject\app\Models\Regform_model.php(16): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\clgproject\app\Controllers\Regform.php(91): App\Models\Regform_model->createUser(Array, Array)
#6 C:\xampp\htdocs\clgproject\system\CodeIgniter.php(919): App\Controllers\Regform->index()
#7 C:\xampp\htdocs\clgproject\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Regform))
#8 C:\xampp\htdocs\clgproject\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\clgproject\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-25 18:42:41 --> Duplicate entry '225' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `stdbasicinfo2` (`sname`, `adno`, `regno`, `gender`, `dob`, `adyear`, `program`, `adress`, `nation`, `state1`, `district`, `pincode`, `adhaar`, `cast`, `religion`, `income`, `guardian`, `guardianrel`, `guardiancon`, `email1`, `phno`) VALUES (:sname:, :adno:, :regno:, :gender:, :dob:, :adyear:, :program:, :adress:, :nation:, :state1:, :district:, :pincode:, :adhaar:, :cast:, :religion:, :income:, :guardian:, :guardianrel:, :guardiancon:, :email1:, :phno:)', [...], false)
 2 APPPATH\Models\Regform_model.php(16): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Regform.php(91): App\Models\Regform_model->createUser([...], [...])
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Regform->index()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Regform))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-25 18:44:28 --> mysqli_sql_exception: Duplicate entry '226' for key 'PRIMARY' in C:\xampp\htdocs\clgproject\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\clgproject\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `st...', 0)
#1 C:\xampp\htdocs\clgproject\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `st...')
#2 C:\xampp\htdocs\clgproject\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `st...')
#3 C:\xampp\htdocs\clgproject\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `st...', Array, false)
#4 C:\xampp\htdocs\clgproject\app\Models\Regform_model.php(16): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\clgproject\app\Controllers\Regform.php(91): App\Models\Regform_model->createUser(Array, Array)
#6 C:\xampp\htdocs\clgproject\system\CodeIgniter.php(919): App\Controllers\Regform->index()
#7 C:\xampp\htdocs\clgproject\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Regform))
#8 C:\xampp\htdocs\clgproject\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\clgproject\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-25 18:44:28 --> Duplicate entry '226' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `stdbasicinfo2` (`sname`, `adno`, `regno`, `gender`, `dob`, `adyear`, `program`, `adress`, `nation`, `state1`, `district`, `pincode`, `adhaar`, `cast`, `religion`, `income`, `guardian`, `guardianrel`, `guardiancon`, `email1`, `phno`) VALUES (:sname:, :adno:, :regno:, :gender:, :dob:, :adyear:, :program:, :adress:, :nation:, :state1:, :district:, :pincode:, :adhaar:, :cast:, :religion:, :income:, :guardian:, :guardianrel:, :guardiancon:, :email1:, :phno:)', [...], false)
 2 APPPATH\Models\Regform_model.php(16): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\Regform.php(91): App\Models\Regform_model->createUser([...], [...])
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Regform->index()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Regform))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
